src/panels/config/js/trigger/trigger_edit.tsx

```diff
@@ -41,25 +45,34 @@ export default class TriggerEdit extends Component<any> {
     super();
 
     this.typeChanged = this.typeChanged.bind(this);
+    this.onYamlChange = this.onYamlChange.bind(this);
   }
 
-  public render({ index, trigger, onChange, hass, localize }) {
+  public render({ index, trigger, onChange, hass, localize, yaml }) {
     // tslint:disable-next-line: variable-name
     const Comp = TYPES[trigger.platform];
     const selected = OPTIONS.indexOf(trigger.platform);
 
-    if (!Comp) {
+    if (yaml || !Comp) {
```

Comment:
why check for `yaml` ? If there is no `yaml` it will now try to render `<Comp />`

Reply:
I made the name clearer, it is whether the editor is in YAML mode, so if it is false it should render `<Comp />`